packages/manager/apps/dedicated/client/app/managedBaremetal/datacenter/host/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "managed_baremetal_datacenters_datacenter_host": "Hosts"
```

Comment:
Can we use an existing translation id?

Reply:
Same response as here https://github.com/ovh/manager/pull/4078#discussion_r529532968